### PR TITLE
Pa 2436

### PR DESCRIPTION
**Pull request by devinleighsmith**
display a warning message when the land classification is changed.

Not sure if this needs to display during the project workflow as well.

### CODE REVIEW

File: frontend/src/features/mapSideBar/SidebarContents/subforms/LandUsageForm.tsx [hunk 1]
```diff
@@ -54,6 +66,15 @@ export const LandUsageForm = <T extends any>(props: ILandUsageProps & FormikProp
...
         />
       </Row>
+      <GenericModal
+        title="Land Classification Changed"
+        message="Remember to update any buildings to this new classification as well."
+        display={showSurplusActiveWarning}
+        setDisplay={setShowSurplusActiveWarning}
+        okButtonText="Ok"
+        handleOk={() => setShowSurplusActiveWarning(false)}
+        handleCancel={() => setShowSurplusActiveWarning(false)}
```
Comment (agalbrai) [inline frontend/src/features/mapSideBar/SidebarContents/subforms/LandUsageForm.tsx:76]: Does the modal need an "Ok" and a "cancel" button?

**Review by agalbrai — APPROVED** [review]

File: frontend/src/features/mapSideBar/SidebarContents/subforms/LandUsageForm.tsx (hunk 1 above)
Comment (Fosol) [reply frontend/src/features/mapSideBar/SidebarContents/subforms/LandUsageForm.tsx:76]: It is just informational, should just be a `Close` button.

File: maintenance/Caddyfile [hunk 2]
```diff
@@ -1,16 +1,19 @@
-0.0.0.0:2015
-
-root /var/www/html
-
-log stdout
-
-errors stdout
+{
+    storage file_system {
```
Comment (Fosol) [inline maintenance/Caddyfile:2]: Mixed PR

File: frontend/src/features/mapSideBar/SidebarContents/subforms/LandUsageForm.tsx (hunk 1 above)
Comment (devinleighsmith) [reply frontend/src/features/mapSideBar/SidebarContents/subforms/LandUsageForm.tsx:76]: it doesn't have a cancel button, but the handleCancel function is used by the x icon.

Comment (devinleighsmith) [reply frontend/src/features/mapSideBar/SidebarContents/subforms/LandUsageForm.tsx:76]: it is also used when the user clicks in any space outside the modal.

**Review by Fosol — APPROVED** [review]

**Pull request merged**